cluster/local/provider.Tiltfile

```diff
@@ -0,0 +1,196 @@
+# 2. build and deploy controller manager
+def build_provider(name, provider, settings):
+    _generate_crds(provider, settings)
+    _build_controller(name, provider, settings)
+
+# Generate and apply crossplane provider's CRDs
+def _generate_crds(provider, settings):
+    context = provider.get('context')
+
+    # Prefix each dependency with context so Tilt can watch the correct paths
+    # for changes. At the same time keep track of 'zz_generated.*' files to
+    # ignore (this is needed to prevent being stuck in an infinite loop.)
+    crd_deps = []
+    ignore_deps = []
+    for d in provider.get('crd_deps', []):
+        crd_deps.append(context + '/' + d)
+        ignore_deps.append(context + '/' + d + '/**/zz_generated.*')
+        
+    # Generate crossplane provider CRDs out of API definition
+    local_resource(
+        'gen-crds-' + provider['short_name'],
+        'cd ' + context + ' && make generate',
+        deps = crd_deps,
+        ignore = ignore_deps
+    )
+
+    # Find all the CRDs manifests files
+    crds_folder = context + '/' + provider.get('crds_folder', 'package/crds')
+    files = listdir(crds_folder)
+
+    # Extract name of the CRDs to show on Tilt UI
+    crds_name = []
+    for f in files:
+        crds_name.append(read_yaml(f)['metadata']['name'] + ':customresourcedefinition')
+
+    # Apply CRD manifests
+    k8s_yaml(files)
+    k8s_resource(
+        new_name = 'crds-' + provider['short_name'],
+        objects = crds_name,
+        resource_deps = settings.get('resource_deps')
+    )
+
+# Build crossplane provider binary and docker image. Binary is built for
+# linux-amd64 (to be only used inside a container.) The binary is stored
+# in crossplane cloned folder at _output/tilt/
+def _build_controller(name, provider, settings):
+    context = provider.get('context')
+    go_main = provider.get('go_main')
+    core_path = settings.get('core_path')
+
+    # Prefix each live reload dependency with context so Tilt
+    # can watch the correct paths for changes
+    cmd_deps = []
+    ignore_deps = []
+    for d in provider.get('cmd_deps', []):
+        cmd_deps.append(context + '/' + d)
+        ignore_deps.append(context + '/' + d + '/**/zz_generated.*')
+
+    # Build crossplane provider
+    build_output = '_output/tilt/' + name
+    local_resource(
+        'build-' + provider['short_name'],
+        'cd ' + context + ' && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ' + core_path + '/' + build_output + ' ' + go_main,
+        deps = cmd_deps,
+        ignore = ignore_deps
+    )
+
+    # Build crossplane provider docker image
+    dockerfile_contents = '\n'.join([
+        'FROM alpine:3.7',
+        'RUN apk --no-cache add ca-certificates bash',
+        'ADD ' + build_output + ' /usr/local/bin/crossplane-provider',
+        # TODO(khos2ow): change this URL right before
+        # merging the PR in crossplane/crossplane!
+        #
+        # https://github.com/crossplane/crossplane/pull/1925
```

Comment:
@khos2ow does this need to be changed?

Reply:
Yes, they need to be changed to `https://raw.githubusercontent.com/crossplane/crossplane/master/...` right before merge. The reason for current one is the make the functionality _workable_ for testing, demo, or example usage in [provider-digitalocean](https://github.com/crossplane-contrib/provider-digitalocean/blob/main/Tiltfile#L17).